### ISSUE TITLE
Login.html

### ISSUE THREAD

**Issue by pallavikotturu** [pallavikotturu/HW8#2]
<!DOCTYPE html>

<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
<head>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.css" />
    <link rel="stylesheet" href="https://ajax.aspnetcdn.com/ajax/bootstrap/3.3.6/css/bootstrap.min.css"
    <title>Login</title>
</head>
<body>
    <div class="navbar navbar-inverse navbar-fixed-top">
        <div class="container">
            <div class="navbar-header">
                <button button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
                    <span class="sr-only">Toggle navigation</span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                </button>

                <img src="Images/Logo.PNG" class="ui-icon-image" height="50" width="50" />
            </div>

            <div id="navbar" class="collapse navbar-collapse navbar-responsive-collapse">
                <ul class="nav navbar-nav">
                    <li><a href="Home.html">Home</a></li>
                    <li class="active"><a href="Login.html">Login</a></li>
                    <li><a href="AboutYou.html">About You</a></li>
                </ul>
            </div>

        </div>
    </div>

    <div class="container" style="margin-top: 5%">
        <div class="col-md-4 col-md-offset-4">
            <div class="panel panel-default">
                <div class="panel-heading"><h1 class="panel-title"><strong>Log In </strong></h1></div>
                <div class="panel-body">
                    <form role="form">
                        <div class="form-group">
                            <label for="exampleInputEmail1">AccessNet Name</label>
                            <input type="email" class="form-control" id="AccessNetID" placeholder="Enter AccessNet Name">
                        </div>
                        <div class="form-group">
                            <label for="exampleInputPassword1">Password</label>
                            <input type="password" class="form-control" id="Password" placeholder="Password">
                        </div>
                        <button type="submit" class="btn btn-sm btn-submit">Log in</button>
                    </form>
                </div>
            </div>
        </div>

</body>
</html>
